draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2081,24 +2108,11 @@ uCDN-A                         dCDN-B                         dCDN-C
            is released for viewing. A scheduled operation enables uCDN to direct dCDN in what
            time frame to execute the trigger.
         </t>
-        <t>
-           uCDN may wish to schedule a trigger such that dCDN will execute it in local time,
-           as it is measured in each region. For example, uCDN may wish dCDN to pull the content
-           at off-peak hours, between 2AM-4AM, however, as a CDN is distributed across multiple time
-           zones, the UTC definition of 2AM depends on the actual location.
-        </t>
-        <t>We define two alternatives for localized scheduling:<list style="symbols">
-           <t>Regional schedule: When used in conjunction with the Location Policy defined
+	<t>We also support regional scheduling, so when used in conjunction with the Location Policy defined
```

Comment:
This is generally good, but it seems to be a bit weird to use "we" in an RFC.  Maybe "This specification" or something like that?

Reply:
fixed, plus edited another occurrence of "we" in the intro around error codes